2013/11/21 23:58:52 [error] [exception.CException] exception 'CException' with message 'CAssetManager.basePath "R:\home\roxifi\www\assets" is invalid. Please make sure the directory exists and is writable by the Web server process.' in R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CAssetManager.php:138
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CAssetManager.php(123): CAssetManager->setBasePath('R:\home\roxifi\...')
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CAssetManager.php(217): CAssetManager->getBasePath()
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\gii\GiiModule.php(143): CAssetManager->publish('R:\home\roxifi\...')
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CComponent.php(111): GiiModule->getAssetsUrl()
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CModule.php(105): CComponent->__get('assetsUrl')
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\gii\views\layouts\main.php(5): CModule->__get('assetsUrl')
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(126): require('R:\home\roxifi\...')
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(95): CBaseController->renderInternal('R:\home\roxifi\...', Array, true)
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('R:\home\roxifi\...', Array, true)
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div class="con...')
#10 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div class="con...')
#11 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(206): COutputProcessor->run()
#12 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#13 R:\home\roxifi\yii-1.1.14.f0fee9\framework\gii\views\layouts\column1.php(7): CBaseController->endContent()
#14 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(126): require('R:\home\roxifi\...')
#15 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(95): CBaseController->renderInternal('R:\home\roxifi\...', Array, true)
#16 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(784): CBaseController->renderFile('R:\home\roxifi\...', Array, true)
#17 R:\home\roxifi\yii-1.1.14.f0fee9\framework\gii\controllers\DefaultController.php(47): CController->render('login', Array)
#18 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(49): DefaultController->actionLogin()
#19 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#20 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#21 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#22 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('login')
#23 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/default/log...')
#24 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#25 R:\home\roxifi\www\index.php(13): CApplication->run()
#26 {main}
REQUEST_URI=/gii/default/login
---
2013/11/22 21:50:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index".' in R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php:483
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(270): CController->missingAction('')
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('Friends')
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 R:\home\roxifi\www\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/Friends
HTTP_REFERER=http://roxifi/gii/controller/index
---
2013/11/22 22:10:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin/userfields".' in R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php:286
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('admin/userfield...')
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 R:\home\roxifi\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/admin/userfields
---
2013/11/22 23:20:58 [error] [exception.CException] exception 'CException' with message 'Property "Users.username" is not defined.' in R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CComponent.php:130
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\db\ar\CActiveRecord.php(145): CComponent->__get('username')
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\helpers\CHtml.php(2529): CActiveRecord->__get('username')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\helpers\CHtml.php(2252): CHtml::resolveValue(Object(Users), 'username')
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\helpers\CHtml.php(1434): CHtml::activeInputField('text', Object(Users), 'username', Array)
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\widgets\CActiveForm.php(725): CHtml::activeTextField(Object(Users), 'username', Array)
#5 R:\home\roxifi\www\protected\views\site\register.php(29): CActiveForm->textField(Object(Users), 'username')
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(126): require('R:\home\roxifi\...')
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(95): CBaseController->renderInternal('R:\home\roxifi\...', Array, true)
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(869): CBaseController->renderFile('R:\home\roxifi\...', Array, true)
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(782): CController->renderPartial('register', Array, true)
#10 R:\home\roxifi\www\protected\controllers\SiteController.php(121): CController->render('register', Array)
#11 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(49): SiteController->actionRegister()
#12 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('register')
#16 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('site/register')
#17 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 R:\home\roxifi\www\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/site/register
---
2013/11/22 23:21:11 [error] [exception.CException] exception 'CException' with message 'Property "Users.rememberMe" is not defined.' in R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CComponent.php:130
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\db\ar\CActiveRecord.php(145): CComponent->__get('rememberMe')
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\helpers\CHtml.php(2529): CActiveRecord->__get('rememberMe')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\helpers\CHtml.php(1743): CHtml::resolveValue(Object(Users), 'rememberMe')
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\widgets\CActiveForm.php(831): CHtml::activeCheckBox(Object(Users), 'rememberMe', Array)
#4 R:\home\roxifi\www\protected\views\site\register.php(43): CActiveForm->checkBox(Object(Users), 'rememberMe')
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(126): require('R:\home\roxifi\...')
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(95): CBaseController->renderInternal('R:\home\roxifi\...', Array, true)
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(869): CBaseController->renderFile('R:\home\roxifi\...', Array, true)
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(782): CController->renderPartial('register', Array, true)
#9 R:\home\roxifi\www\protected\controllers\SiteController.php(121): CController->render('register', Array)
#10 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(49): SiteController->actionRegister()
#11 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('register')
#15 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('site/register')
#16 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#17 R:\home\roxifi\www\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/site/register
---
2013/11/22 23:33:52 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "isNewRecord".' in R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CComponent.php:266
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\db\ar\CActiveRecord.php(227): CComponent->__call('isNewRecord', Array)
#1 [internal function]: CActiveRecord->__call('isNewRecord', Array)
#2 R:\home\roxifi\www\protected\models\Users.php(134): Users->isNewRecord()
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CModel.php(156): Users->beforeValidate()
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\db\ar\CActiveRecord.php(805): CModel->validate(NULL)
#5 R:\home\roxifi\www\protected\controllers\SiteController.php(119): CActiveRecord->save()
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(49): SiteController->actionRegister()
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('register')
#11 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('site/register')
#12 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 R:\home\roxifi\www\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/site/register
HTTP_REFERER=http://roxifi/site/register
---
2013/11/23 01:03:04 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "register_success".' in R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php:878
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(782): CController->renderPartial('register_succes...', NULL, true)
#1 R:\home\roxifi\www\protected\controllers\SiteController.php(122): CController->render('register_succes...')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(49): SiteController->actionRegister()
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('register')
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('site/register')
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 R:\home\roxifi\www\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/site/register
HTTP_REFERER=http://roxifi/site/register
---
2013/11/23 01:21:17 [error] [exception.CException] exception 'CException' with message 'В классе Users и его поведениях не найден метод или замыкание с именем "validatePassword".' in R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CComponent.php:266
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\db\ar\CActiveRecord.php(227): CComponent->__call('validatePasswor...', Array)
#1 [internal function]: CActiveRecord->__call('validatePasswor...', Array)
#2 R:\home\roxifi\www\protected\components\UserIdentity.php(23): Users->validatePassword('159631')
#3 R:\home\roxifi\www\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#7 R:\home\roxifi\www\protected\controllers\SiteController.php(94): CModel->validate()
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('login')
#13 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#14 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 R:\home\roxifi\www\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/site/login
HTTP_REFERER=http://roxifi/site/login
---
2013/11/23 02:05:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "profile/view".' in R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php:286
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('profile/view')
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 R:\home\roxifi\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/profile/7
HTTP_REFERER=http://roxifi/
---
2013/11/23 02:05:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "profile/view".' in R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php:286
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('profile/view')
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 R:\home\roxifi\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/profile/7
HTTP_REFERER=http://roxifi/
---
2013/11/23 02:09:41 [error] [exception.CHttpException.Профиль не найден] exception 'CHttpException' with message '404' in R:\home\roxifi\www\protected\models\Users.php:173
Stack trace:
#0 R:\home\roxifi\www\protected\controllers\UsersController.php(15): Users->findByPk('2')
#1 [internal function]: UsersController->actionProfile('2')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('profile')
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('users/profile')
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 R:\home\roxifi\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/profile/2
---
2013/11/23 02:10:28 [error] [exception.CHttpException.Профиль не найден] exception 'CHttpException' with message '404' in R:\home\roxifi\www\protected\models\Users.php:173
Stack trace:
#0 R:\home\roxifi\www\protected\controllers\UsersController.php(15): Users->findByPk('2')
#1 [internal function]: UsersController->actionProfile('2')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('profile')
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('users/profile')
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 R:\home\roxifi\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/profile/2
---
2013/11/23 02:10:40 [error] [exception.CHttpException.Профиль не найден] exception 'CHttpException' with message '404' in R:\home\roxifi\www\protected\models\Users.php:173
Stack trace:
#0 R:\home\roxifi\www\protected\controllers\UsersController.php(15): Users->findByPk('2')
#1 [internal function]: UsersController->actionProfile('2')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('profile')
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('users/profile')
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 R:\home\roxifi\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/profile/2
---
2013/11/23 02:10:41 [error] [exception.CHttpException.Профиль не найден] exception 'CHttpException' with message '404' in R:\home\roxifi\www\protected\models\Users.php:173
Stack trace:
#0 R:\home\roxifi\www\protected\controllers\UsersController.php(15): Users->findByPk('2')
#1 [internal function]: UsersController->actionProfile('2')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('profile')
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('users/profile')
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 R:\home\roxifi\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/profile/2
---
2013/11/23 02:11:52 [error] [exception.CHttpException.Профиль не найден] exception 'CHttpException' with message '404' in R:\home\roxifi\www\protected\models\Users.php:173
Stack trace:
#0 R:\home\roxifi\www\protected\controllers\UsersController.php(15): Users->findByPk('2')
#1 [internal function]: UsersController->actionProfile('2')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('profile')
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('users/profile')
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 R:\home\roxifi\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/profile/2
---
2013/11/23 02:12:09 [error] [exception.CHttpException.Профиль не найден] exception 'CHttpException' with message '404' in R:\home\roxifi\www\protected\models\Users.php:173
Stack trace:
#0 R:\home\roxifi\www\protected\controllers\UsersController.php(15): Users->findByPk('2')
#1 [internal function]: UsersController->actionProfile('2')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('profile')
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('users/profile')
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 R:\home\roxifi\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/profile/2
---
2013/11/23 02:12:13 [error] [exception.CHttpException.Профиль не найден] exception 'CHttpException' with message '404' in R:\home\roxifi\www\protected\models\Users.php:173
Stack trace:
#0 R:\home\roxifi\www\protected\controllers\UsersController.php(15): Users->findByPk('2')
#1 [internal function]: UsersController->actionProfile('2')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('profile')
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('users/profile')
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 R:\home\roxifi\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/profile/2
---
2013/11/23 02:12:51 [error] [exception.CHttpException.Профиль не найден] exception 'CHttpException' with message '404' in R:\home\roxifi\www\protected\models\Users.php:173
Stack trace:
#0 R:\home\roxifi\www\protected\controllers\UsersController.php(15): Users->findByPk('2')
#1 [internal function]: UsersController->actionProfile('2')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('profile')
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('users/profile')
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 R:\home\roxifi\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/profile/2
---
2013/11/23 02:12:52 [error] [exception.CHttpException.Профиль не найден] exception 'CHttpException' with message '404' in R:\home\roxifi\www\protected\models\Users.php:173
Stack trace:
#0 R:\home\roxifi\www\protected\controllers\UsersController.php(15): Users->findByPk('2')
#1 [internal function]: UsersController->actionProfile('2')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('profile')
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('users/profile')
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 R:\home\roxifi\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/profile/2
---
2013/11/23 02:13:02 [error] [exception.CHttpException.Профиль не найден] exception 'CHttpException' with message '404' in R:\home\roxifi\www\protected\models\Users.php:173
Stack trace:
#0 R:\home\roxifi\www\protected\controllers\UsersController.php(15): Users->findByPk('2')
#1 [internal function]: UsersController->actionProfile('2')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('profile')
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('users/profile')
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 R:\home\roxifi\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/profile/2
---
2013/11/23 02:13:12 [error] [exception.CHttpException.Профиль не найден] exception 'CHttpException' with message '404' in R:\home\roxifi\www\protected\models\Users.php:173
Stack trace:
#0 R:\home\roxifi\www\protected\controllers\UsersController.php(15): Users->findByPk('2')
#1 [internal function]: UsersController->actionProfile('2')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('profile')
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('users/profile')
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 R:\home\roxifi\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/profile/2
---
2013/11/23 02:13:38 [error] [exception.CHttpException.Профиль не найден] exception 'CHttpException' with message '404' in R:\home\roxifi\www\protected\models\Users.php:173
Stack trace:
#0 R:\home\roxifi\www\protected\controllers\UsersController.php(15): Users->findByPk('2')
#1 [internal function]: UsersController->actionProfile('2')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('profile')
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('users/profile')
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 R:\home\roxifi\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/profile/2
---
2013/11/23 02:13:44 [error] [exception.CHttpException.Профиль не найден] exception 'CHttpException' with message '404' in R:\home\roxifi\www\protected\models\Users.php:173
Stack trace:
#0 R:\home\roxifi\www\protected\controllers\UsersController.php(15): Users->findByPk('2')
#1 [internal function]: UsersController->actionProfile('2')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('profile')
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('users/profile')
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 R:\home\roxifi\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/profile/2
---
2013/11/23 02:14:01 [error] [exception.CHttpException.Профиль не найден] exception 'CHttpException' with message '404' in R:\home\roxifi\www\protected\models\Users.php:173
Stack trace:
#0 R:\home\roxifi\www\protected\controllers\UsersController.php(15): Users->findByPk('2')
#1 [internal function]: UsersController->actionProfile('2')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('profile')
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('users/profile')
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 R:\home\roxifi\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/profile/2
---
2013/11/23 02:14:04 [error] [exception.CHttpException.Профиль не найден] exception 'CHttpException' with message '404' in R:\home\roxifi\www\protected\models\Users.php:173
Stack trace:
#0 R:\home\roxifi\www\protected\controllers\UsersController.php(15): Users->findByPk('2')
#1 [internal function]: UsersController->actionProfile('2')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('profile')
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('users/profile')
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 R:\home\roxifi\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/profile/2
---
2013/11/23 02:14:08 [error] [exception.CHttpException.Профиль не найден] exception 'CHttpException' with message '404' in R:\home\roxifi\www\protected\models\Users.php:173
Stack trace:
#0 R:\home\roxifi\www\protected\controllers\UsersController.php(15): Users->findByPk('2')
#1 [internal function]: UsersController->actionProfile('2')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('profile')
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('users/profile')
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 R:\home\roxifi\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/profile/2
---
2013/11/23 02:14:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Профиль не найден' in R:\home\roxifi\www\protected\models\Users.php:173
Stack trace:
#0 R:\home\roxifi\www\protected\controllers\UsersController.php(15): Users->findByPk('2')
#1 [internal function]: UsersController->actionProfile('2')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('profile')
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('users/profile')
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 R:\home\roxifi\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/profile/2
---
2013/11/23 02:17:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Профиль не найден' in R:\home\roxifi\www\protected\models\Users.php:173
Stack trace:
#0 R:\home\roxifi\www\protected\controllers\UsersController.php(15): Users->findByPk('2')
#1 [internal function]: UsersController->actionProfile('2')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('profile')
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('users/profile')
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 R:\home\roxifi\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/profile/2
---
2013/11/23 02:18:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Профиль не найден' in R:\home\roxifi\www\protected\models\Users.php:173
Stack trace:
#0 R:\home\roxifi\www\protected\controllers\UsersController.php(15): Users->findByPk('2')
#1 [internal function]: UsersController->actionProfile('2')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('profile')
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('users/profile')
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 R:\home\roxifi\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/profile/2
---
2013/11/23 02:18:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Профиль не найден' in R:\home\roxifi\www\protected\models\Users.php:173
Stack trace:
#0 R:\home\roxifi\www\protected\controllers\UsersController.php(15): Users->findByPk('2')
#1 [internal function]: UsersController->actionProfile('2')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('profile')
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('users/profile')
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 R:\home\roxifi\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/profile/2
---
2013/11/23 02:39:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Профиль не найден' in R:\home\roxifi\www\protected\models\Users.php:173
Stack trace:
#0 R:\home\roxifi\www\protected\controllers\UsersController.php(15): Users->findByPk('2')
#1 [internal function]: UsersController->actionProfile('2')
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('profile')
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('users/profile')
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 R:\home\roxifi\www\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/profile/2
HTTP_REFERER=http://roxifi/profile/7
---
2013/11/23 02:43:44 [error] [exception.CDbException] exception 'CDbException' with message 'Отношение "" не было определено в active record классе "Users".' in R:\home\roxifi\yii-1.1.14.f0fee9\framework\db\ar\CActiveFinder.php:226
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\db\ar\CActiveFinder.php(296): CActiveFinder->buildJoinTree(Object(CJoinElement), '')
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\db\ar\CActiveFinder.php(48): CActiveFinder->buildJoinTree(Object(CJoinElement), Array)
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\db\ar\CActiveRecord.php(909): CActiveFinder->__construct(Object(Users), Array)
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\db\ar\CActiveRecord.php(1355): CActiveRecord->getActiveFinder(Array)
#4 R:\home\roxifi\www\protected\models\Users.php(171): CActiveRecord->query(Object(CDbCriteria))
#5 R:\home\roxifi\www\protected\controllers\UsersController.php(15): Users->findByPk('7')
#6 [internal function]: UsersController->actionProfile('7')
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('profile')
#13 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('users/profile')
#14 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 R:\home\roxifi\www\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/profile/7
---
2013/11/23 02:47:01 [error] [php] Object of class WallRecords could not be converted to string (R:\home\roxifi\www\protected\views\users\Profile.php:32)
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(869): UsersController->renderFile()
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(782): UsersController->renderPartial()
#2 R:\home\roxifi\www\protected\controllers\UsersController.php(15): UsersController->render()
#3 unknown(0): UsersController->actionProfile()
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): UsersController->runAction()
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): UsersController->runActionWithFilters()
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): UsersController->run()
#10 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 R:\home\roxifi\www\index.php(13): CWebApplication->run()
REQUEST_URI=/profile/7
in R:\home\roxifi\www\protected\views\users\Profile.php (32)
in R:\home\roxifi\www\protected\controllers\UsersController.php (15)
in R:\home\roxifi\www\index.php (13)
2013/11/23 16:42:50 [error] [php] Error parsing lang/ru.conf on line 1
 (R:\home\roxifi\www\protected\models\Language.php:28)
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(126): require()
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(95): SiteController->renderInternal()
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(869): SiteController->renderFile()
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(782): SiteController->renderPartial()
#4 R:\home\roxifi\www\protected\controllers\SiteController.php(32): SiteController->render()
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): SiteController->runAction()
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): SiteController->runActionWithFilters()
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): SiteController->run()
#10 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 R:\home\roxifi\www\index.php(13): CWebApplication->run()
REQUEST_URI=/
in R:\home\roxifi\www\protected\models\Language.php (28)
in R:\home\roxifi\www\protected\views\site\index.php (1)
in R:\home\roxifi\www\protected\controllers\SiteController.php (32)
2013/11/23 16:43:40 [error] [php] Error parsing lang/ru.conf on line 1
 (R:\home\roxifi\www\protected\models\Language.php:28)
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(126): require()
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(95): SiteController->renderInternal()
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(869): SiteController->renderFile()
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(782): SiteController->renderPartial()
#4 R:\home\roxifi\www\protected\controllers\SiteController.php(32): SiteController->render()
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): SiteController->runAction()
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): SiteController->runActionWithFilters()
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): SiteController->run()
#10 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 R:\home\roxifi\www\index.php(13): CWebApplication->run()
REQUEST_URI=/
in R:\home\roxifi\www\protected\models\Language.php (28)
in R:\home\roxifi\www\protected\views\site\index.php (1)
in R:\home\roxifi\www\protected\controllers\SiteController.php (32)
2013/11/23 16:45:19 [error] [php] Error parsing lang/ru.conf on line 1
 (R:\home\roxifi\www\protected\models\Language.php:28)
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(126): require()
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(95): SiteController->renderInternal()
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(869): SiteController->renderFile()
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(782): SiteController->renderPartial()
#4 R:\home\roxifi\www\protected\controllers\SiteController.php(32): SiteController->render()
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): SiteController->runAction()
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): SiteController->runActionWithFilters()
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): SiteController->run()
#10 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 R:\home\roxifi\www\index.php(13): CWebApplication->run()
REQUEST_URI=/
in R:\home\roxifi\www\protected\models\Language.php (28)
in R:\home\roxifi\www\protected\views\site\index.php (1)
in R:\home\roxifi\www\protected\controllers\SiteController.php (32)
2013/11/23 16:45:55 [error] [php] Error parsing lang/ru.conf on line 1
 (R:\home\roxifi\www\protected\models\Language.php:28)
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(126): require()
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(95): SiteController->renderInternal()
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(869): SiteController->renderFile()
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(782): SiteController->renderPartial()
#4 R:\home\roxifi\www\protected\controllers\SiteController.php(32): SiteController->render()
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): SiteController->runAction()
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): SiteController->runActionWithFilters()
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): SiteController->run()
#10 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 R:\home\roxifi\www\index.php(13): CWebApplication->run()
REQUEST_URI=/
in R:\home\roxifi\www\protected\models\Language.php (28)
in R:\home\roxifi\www\protected\views\site\index.php (1)
in R:\home\roxifi\www\protected\controllers\SiteController.php (32)
2013/11/23 16:58:30 [error] [php] Error parsing lang/ru.conf on line 1
 (R:\home\roxifi\www\protected\models\Language.php:28)
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(126): require()
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(95): SiteController->renderInternal()
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(869): SiteController->renderFile()
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(782): SiteController->renderPartial()
#4 R:\home\roxifi\www\protected\controllers\SiteController.php(32): SiteController->render()
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): SiteController->runAction()
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): SiteController->runActionWithFilters()
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): SiteController->run()
#10 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 R:\home\roxifi\www\index.php(13): CWebApplication->run()
REQUEST_URI=/
in R:\home\roxifi\www\protected\models\Language.php (28)
in R:\home\roxifi\www\protected\views\site\index.php (1)
in R:\home\roxifi\www\protected\controllers\SiteController.php (32)
2013/11/23 16:58:31 [error] [php] Error parsing lang/ru.conf on line 1
 (R:\home\roxifi\www\protected\models\Language.php:28)
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(126): require()
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(95): SiteController->renderInternal()
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(869): SiteController->renderFile()
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(782): SiteController->renderPartial()
#4 R:\home\roxifi\www\protected\controllers\SiteController.php(32): SiteController->render()
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): SiteController->runAction()
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): SiteController->runActionWithFilters()
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): SiteController->run()
#10 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 R:\home\roxifi\www\index.php(13): CWebApplication->run()
REQUEST_URI=/
in R:\home\roxifi\www\protected\models\Language.php (28)
in R:\home\roxifi\www\protected\views\site\index.php (1)
in R:\home\roxifi\www\protected\controllers\SiteController.php (32)
2013/11/23 16:58:55 [error] [php] Error parsing lang/ru.conf on line 1
 (R:\home\roxifi\www\protected\models\Language.php:28)
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(126): require()
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(95): SiteController->renderInternal()
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(869): SiteController->renderFile()
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(782): SiteController->renderPartial()
#4 R:\home\roxifi\www\protected\controllers\SiteController.php(32): SiteController->render()
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): SiteController->runAction()
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): SiteController->runActionWithFilters()
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): SiteController->run()
#10 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 R:\home\roxifi\www\index.php(13): CWebApplication->run()
REQUEST_URI=/
in R:\home\roxifi\www\protected\models\Language.php (28)
in R:\home\roxifi\www\protected\views\site\index.php (1)
in R:\home\roxifi\www\protected\controllers\SiteController.php (32)
2013/11/23 16:59:01 [error] [php] Error parsing lang/ru.conf on line 1
 (R:\home\roxifi\www\protected\models\Language.php:28)
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(126): require()
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(95): SiteController->renderInternal()
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(869): SiteController->renderFile()
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(782): SiteController->renderPartial()
#4 R:\home\roxifi\www\protected\controllers\SiteController.php(32): SiteController->render()
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): SiteController->runAction()
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): SiteController->runActionWithFilters()
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): SiteController->run()
#10 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 R:\home\roxifi\www\index.php(13): CWebApplication->run()
REQUEST_URI=/
in R:\home\roxifi\www\protected\models\Language.php (28)
in R:\home\roxifi\www\protected\views\site\index.php (1)
in R:\home\roxifi\www\protected\controllers\SiteController.php (32)
2013/11/23 16:59:20 [error] [php] Error parsing lang/ru.conf on line 3
 (R:\home\roxifi\www\protected\models\Language.php:28)
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(126): require()
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(95): SiteController->renderInternal()
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(869): SiteController->renderFile()
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(782): SiteController->renderPartial()
#4 R:\home\roxifi\www\protected\controllers\SiteController.php(32): SiteController->render()
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): SiteController->runAction()
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): SiteController->runActionWithFilters()
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): SiteController->run()
#10 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 R:\home\roxifi\www\index.php(13): CWebApplication->run()
REQUEST_URI=/
in R:\home\roxifi\www\protected\models\Language.php (28)
in R:\home\roxifi\www\protected\views\site\index.php (1)
in R:\home\roxifi\www\protected\controllers\SiteController.php (32)
2013/11/23 16:59:22 [error] [php] Error parsing lang/ru.conf on line 3
 (R:\home\roxifi\www\protected\models\Language.php:28)
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(126): require()
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(95): SiteController->renderInternal()
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(869): SiteController->renderFile()
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(782): SiteController->renderPartial()
#4 R:\home\roxifi\www\protected\controllers\SiteController.php(32): SiteController->render()
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): SiteController->runAction()
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): SiteController->runActionWithFilters()
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): SiteController->run()
#10 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 R:\home\roxifi\www\index.php(13): CWebApplication->run()
REQUEST_URI=/
in R:\home\roxifi\www\protected\models\Language.php (28)
in R:\home\roxifi\www\protected\views\site\index.php (1)
in R:\home\roxifi\www\protected\controllers\SiteController.php (32)
2013/11/23 16:59:27 [error] [php] Error parsing lang/ru.conf on line 2
 (R:\home\roxifi\www\protected\models\Language.php:28)
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(126): require()
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(95): SiteController->renderInternal()
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(869): SiteController->renderFile()
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(782): SiteController->renderPartial()
#4 R:\home\roxifi\www\protected\controllers\SiteController.php(32): SiteController->render()
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): SiteController->runAction()
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): SiteController->runActionWithFilters()
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): SiteController->run()
#10 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 R:\home\roxifi\www\index.php(13): CWebApplication->run()
REQUEST_URI=/
in R:\home\roxifi\www\protected\models\Language.php (28)
in R:\home\roxifi\www\protected\views\site\index.php (1)
in R:\home\roxifi\www\protected\controllers\SiteController.php (32)
2013/11/23 17:00:15 [error] [php] Error parsing lang/ru.conf on line 1
 (R:\home\roxifi\www\protected\models\Language.php:28)
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(126): require()
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(95): SiteController->renderInternal()
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(869): SiteController->renderFile()
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(782): SiteController->renderPartial()
#4 R:\home\roxifi\www\protected\controllers\SiteController.php(32): SiteController->render()
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): SiteController->runAction()
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): SiteController->runActionWithFilters()
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): SiteController->run()
#10 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 R:\home\roxifi\www\index.php(13): CWebApplication->run()
REQUEST_URI=/
in R:\home\roxifi\www\protected\models\Language.php (28)
in R:\home\roxifi\www\protected\views\site\index.php (1)
in R:\home\roxifi\www\protected\controllers\SiteController.php (32)
2013/11/23 17:00:16 [error] [php] Error parsing lang/ru.conf on line 1
 (R:\home\roxifi\www\protected\models\Language.php:28)
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(126): require()
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(95): SiteController->renderInternal()
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(869): SiteController->renderFile()
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(782): SiteController->renderPartial()
#4 R:\home\roxifi\www\protected\controllers\SiteController.php(32): SiteController->render()
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): SiteController->runAction()
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): SiteController->runActionWithFilters()
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): SiteController->run()
#10 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 R:\home\roxifi\www\index.php(13): CWebApplication->run()
REQUEST_URI=/
in R:\home\roxifi\www\protected\models\Language.php (28)
in R:\home\roxifi\www\protected\views\site\index.php (1)
in R:\home\roxifi\www\protected\controllers\SiteController.php (32)
2013/11/23 17:00:21 [error] [php] Error parsing lang/ru.conf on line 1
 (R:\home\roxifi\www\protected\models\Language.php:28)
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(126): require()
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(95): SiteController->renderInternal()
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(869): SiteController->renderFile()
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(782): SiteController->renderPartial()
#4 R:\home\roxifi\www\protected\controllers\SiteController.php(32): SiteController->render()
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): SiteController->runAction()
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): SiteController->runActionWithFilters()
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): SiteController->run()
#10 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 R:\home\roxifi\www\index.php(13): CWebApplication->run()
REQUEST_URI=/
in R:\home\roxifi\www\protected\models\Language.php (28)
in R:\home\roxifi\www\protected\views\site\index.php (1)
in R:\home\roxifi\www\protected\controllers\SiteController.php (32)
2013/11/23 17:29:01 [error] [system.db.CDbCommand] Не удалось выполнить CDbCommand::execute(): SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '7' for key 'user_from'. Выполнявшийся SQL-запрос: INSERT INTO `wall_records` (`text`, `user_from`, `user_to`, `time`, `status`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4).
in R:\home\roxifi\www\protected\controllers\UsersController.php (23)
in R:\home\roxifi\www\index.php (13)
2013/11/23 17:29:01 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand не удалось исполнить SQL-запрос: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '7' for key 'user_from'. The SQL statement executed was: INSERT INTO `wall_records` (`text`, `user_from`, `user_to`, `time`, `status`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4)' in R:\home\roxifi\yii-1.1.14.f0fee9\framework\db\CDbCommand.php:358
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\db\ar\CActiveRecord.php(1077): CDbCommand->execute()
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\db\ar\CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 R:\home\roxifi\www\protected\controllers\UsersController.php(23): CActiveRecord->save()
#3 [internal function]: UsersController->actionProfile('7')
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): CController->run('profile')
#10 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('users/profile')
#11 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 R:\home\roxifi\www\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/profile/7
HTTP_REFERER=http://roxifi/profile/7
---
2013/11/23 17:47:14 [error] [php] include(Projects.php) [<a href='function.include'>function.include</a>]: failed to open stream: No such file or directory (R:\home\roxifi\yii-1.1.14.f0fee9\framework\YiiBase.php:427)
Stack trace:
#0 R:\home\roxifi\www\protected\controllers\UsersController.php(32): spl_autoload_call()
#1 unknown(0): UsersController->actionProfile()
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): UsersController->runAction()
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): UsersController->runActionWithFilters()
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): UsersController->run()
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 R:\home\roxifi\www\index.php(13): CWebApplication->run()
REQUEST_URI=/profile/7
in R:\home\roxifi\www\protected\controllers\UsersController.php (32)
in R:\home\roxifi\www\index.php (13)
2013/11/23 17:47:16 [error] [php] include(Projects.php) [<a href='function.include'>function.include</a>]: failed to open stream: No such file or directory (R:\home\roxifi\yii-1.1.14.f0fee9\framework\YiiBase.php:427)
Stack trace:
#0 R:\home\roxifi\www\protected\controllers\UsersController.php(32): spl_autoload_call()
#1 unknown(0): UsersController->actionProfile()
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#3 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): UsersController->runAction()
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): UsersController->runActionWithFilters()
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): UsersController->run()
#8 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 R:\home\roxifi\www\index.php(13): CWebApplication->run()
REQUEST_URI=/profile/7
in R:\home\roxifi\www\protected\controllers\UsersController.php (32)
in R:\home\roxifi\www\index.php (13)
2013/11/23 17:49:42 [error] [php] include(Pager.php) [<a href='function.include'>function.include</a>]: failed to open stream: No such file or directory (R:\home\roxifi\yii-1.1.14.f0fee9\framework\YiiBase.php:427)
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWidgetFactory.php(146): import()
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(146): CWidgetFactory->createWidget()
#2 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(172): UsersController->createWidget()
#3 R:\home\roxifi\www\protected\views\users\Profile.php(42): UsersController->widget()
#4 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(126): require()
#5 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CBaseController.php(95): UsersController->renderInternal()
#6 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(869): UsersController->renderFile()
#7 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(782): UsersController->renderPartial()
#8 R:\home\roxifi\www\protected\controllers\UsersController.php(42): UsersController->render()
#9 unknown(0): UsersController->actionProfile()
#10 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#11 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#12 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(308): CInlineAction->runWithParams()
#13 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(286): UsersController->runAction()
#14 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CController.php(265): UsersController->runActionWithFilters()
#15 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(282): UsersController->run()
#16 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController()
#17 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 R:\home\roxifi\www\index.php(13): CWebApplication->run()
REQUEST_URI=/profile/7
in R:\home\roxifi\www\protected\views\users\Profile.php (42)
in R:\home\roxifi\www\protected\controllers\UsersController.php (42)
in R:\home\roxifi\www\index.php (13)
2013/11/23 20:51:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "favicon.ico".' in R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php:286
Stack trace:
#0 R:\home\roxifi\yii-1.1.14.f0fee9\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 R:\home\roxifi\yii-1.1.14.f0fee9\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 R:\home\roxifi\www\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
